crates/network-monitor/src/lib.rs

```diff
@@ -0,0 +1,241 @@
+use std::collections::HashMap;
+use std::collections::HashSet;
+use std::collections::VecDeque;
+use std::fs;
+use std::future::Future;
+use std::io;
+use std::mem;
+use std::pin::Pin;
+use std::sync::Arc;
+use std::time::Duration;
+
+use camino::*;
+use chrono::{DateTime, TimeDelta, Utc};
+use serde::*;
```

Comment:
issue: I highly discourage using glob imports. There are some exceptions, but we should not decide in an ad-hoc way.